PaLM support (#272)
Oneflow-Inc/libai: projects/PaLM/README.md

<img src="./assets/palm.gif" width="450px"></img>

# Pathways Language Model (PaLM) based on LiBai

> A OneFlow implementation of the model architecture of [Pathways Language Model (PaLM): Scaling to 540 Billion Parameters for Breakthrough Performance](https://ai.googleblog.com/2022/04/pathways-language-model-palm-scaling-to.html) in less than <a href="https://github.com/Oneflow-Inc/libai/tree/main/projects/PaLM/palm_model.py"> 300 lines of a code</a>.

Reviewer: ```suggestion
Implementation of the model architecture of [Pathways Language Model (PaLM): Scaling to 540 Billion Parameters for Breakthrough Performance](https://ai.googleblog.com/2022/04/pathways-language-model-palm-scaling-to.html) with OneFlow in less than <a href="https://github.com/Oneflow-Inc/libai/tree/main/projects/PaLM/palm_model.py"> 300 lines of a code</a>.
```

> We take advantage of [LiBai](https://github.com/hpcaitech/ColossalAI) to exploit multiple parallelism strategies, e.g. data parallelism, tensor parallelism and pipeline parallelism. Besides, some advanced features such as mixed precision and ZeRO come here to help scale the training to multiple GPUs.

Reviewer 2: 这里的链接是不是要换一下~

> You are very welcome to contribute to help us enhance the usability of this project.

Reviewer: ```suggestion
We welcome contributions, to help us enhance the usability of this project.
```

## Install

> 1. Install LiBai following <a href="https://libai.readthedocs.io/en/latest/tutorials/get_started/Installation.html">install instruction</a>.

Reviewer: ```suggestion
1. To install LiBai please follow <a href="https://libai.readthedocs.io/en/latest/tutorials/get_started/Installation.html">install instruction</a>.
```
